crates/snapbox/src/data.rs

```diff
@@ -360,9 +360,40 @@ fn normalize_value_matches(
             *act = substitutions.normalize(act, exp);
         }
         (Array(act), Array(exp)) => {
-            act.iter_mut()
-                .zip(exp)
-                .for_each(|(a, e)| normalize_value_matches(a, e, substitutions));
+            let wildcard = String("{...}".to_string());
+            let mut sections = exp.split(|e| e == &wildcard).peekable();
+            let mut processed = 0;
+            while let Some(section) = sections.next() {
+                // Process all values in the current section
+                if !section.is_empty() {
+                    let slice = &mut act[processed..processed + section.len()];
```

Comment:
To make things clearer, should rename `section` to `expected_subset` and `slice` to `actual_subset`